lib/service/commands/info.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Service
+  module Commands
+    module Info
+      module_function
+
+      TRIGGERS = %w[info i].freeze
+
+      def run(targets, verbose:, json:)
+        return unless ServicesCli.check(targets)
+
+        output = targets.map(&:to_hash)
+
+        if json
+          puts JSON.pretty_generate(output)
+          return
+        end
+
+        output.each do |hash|
+          puts output(hash, verbose: verbose)
+        end
+      end
+
+      def pretty_bool(bool)
+        bool ? "✅" : "❌"
```

Comment:
```suggestion
        return bool if !$stdout.tty? || Homebrew::EnvConfig.no_emoji?
        
        if bool
          "#{Tty.bold}#{Formatter.success("✔")}#{Tty.reset}"
        else
           "#{Tty.bold}#{Formatter.error("✘")}#{Tty.reset}"
        end
```

for consistency with `pretty_installed` and `pretty_uninstalled` used by `brew info` in Homebrew/brew. Might be nice in follow-up PRs to refactor those functions so they can be used here.

Reply:
> Might be nice in follow-up PRs to refactor those functions so they can be used here.

Sounds good, maybe I'll even wait until that PR is released before I merge the service changes 😅